src/server/views/help/cookies.html

```diff
@@ -34,6 +39,33 @@ <h2 class="govuk-heading-m">Essential cookies</h2>
         ]
       ]
     }) }}
+
+    <h2 class="govuk-heading-m">Analytics cookies</h2>
+    <p class="govuk-body">We use Google Analytics cookies to collect information about how you use this form.</p>
+    {{ govukTable({
+      firstCellIsHeader: true,
+      caption: "Analytics cookies we use",
+      head: [
+        { text: "Name" },
+        { text: "Purpose" },
+        { text: "Expires" }
+      ],
+      rows: [
+        [
+          { text: "_ga" },
+          { text: "Used by Google Analytics to help us count how many people visit our forms by tracking if you’ve visited before" },
+          { text: "2 years" }
+        ],
+        [
+          { text: "_ga_" + googleAnalyticsTrackingId },
```

Comment:
Is this definitely correct?

The [GOV.UK Design System uses `ga_[random number]`](https://design-system.service.gov.uk/cookies/)

Reply:
Pretty sure it is. It's certainly not a random number, but there might be a different ID called a 'container ID' we need to look for rather than the tag ID. I'll merge this and verify in the test env.